src/planets/Jupiter.ts

```diff
@@ -42,11 +41,17 @@ export default class Jupiter extends Planet {
         });
     }
 
-    public async getApparentMagnitude(): Promise<number> {
-        const coordsHelio = await this.getHeliocentricEclipticSphericalDateCoordinates();
-        const coordsGeo = await this.getGeocentricEclipticSphericalDateCoordinates();
-        const i = await this.getPhaseAngle();
+    protected calculateApparentMagnitude(
```

Comment:
My idea is to separate the classes and the calculations. So I would like not to do any kind of calculations in Planet Objects.

Reply:
Ok, I see. I've restored the functions in `magnitudeCalc.ts` and they still can be reused here.

The fact that all the calculations are in the `utils` package looks a bit strange. Of course, some of them are properly placed there (like the ones serving for general purpose), but the concrete functions like `getApparentMagnitude*` correspond more to the _core_ functionality of the library. They are not really _utility_ functions.

If you want to keep them separated, I'd suggest placing them in a folder named `calculations`, or closer to the planets definitions. Also, they can be split in separate files.

Reply:
I do like that idea of having a folder `calculations` for each aggregate. This is more a DDD approach.